### PR TITLE
loader: add remark-loader to the list

### LINKED ISSUE #3900: add `remark-loader` to official loaders
__Feature to document__

Need to add `remark-loader` to official loaders for markdown files https://github.com/webpack-contrib/remark-loader

__Additional information__

Any

[ ] I am willing to work on this issue and submit a pull request.

### PR DESCRIPTION
**Pull request by anshumanv**
Closes #3900 

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/loaders/index.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 - [`handlebars-loader`](https://github.com/pcardune/handlebars-loader) Compiles Handlebars to HTML
 - [`markup-inline-loader`](https://github.com/asnowwolf/markup-inline-loader) Inline SVG/MathML files to HTML. It’s useful when applying icon font or applying CSS animation to SVG.
 - [`twig-loader`](https://github.com/zimmo-be/twig-loader) Compiles Twig templates and returns a function
+- [`remark-loader`](https://github.com/webpack-contrib/remark-loader) Load markdown through `remark` with built-in image resolution
```
Comment (evilebottnawi) [inline src/content/loaders/index.md:54]: Just note - it is official loaders, so we should put them in left column here

Comment (anshumanv) [reply src/content/loaders/index.md:54]: gotcha will add

File: repositories/loaders.json [hunk 2]
```diff
@@ -26,5 +26,6 @@
   "webpack-contrib/polymer-webpack-loader",
   "babel/babel-loader",
   "postcss/postcss-loader",
```
Comment (evilebottnawi) [inline repositories/loaders.json:28]: Somebody can create an issue, `postcss/postcss-loader` is now here https://github.com/webpack-contrib/postcss-loader

Comment (anshumanv) [reply repositories/loaders.json:28]: will do

Comment (anshumanv) [reply repositories/loaders.json:28]: Fixed here - https://github.com/webpack/webpack.js.org/pull/3909

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**